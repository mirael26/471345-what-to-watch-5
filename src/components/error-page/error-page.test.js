import React from "react";
import renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import browserHistory from "../../browser-history";

import ErrorPage from "./error-page";

it(`Should ErrorPage render correctly`, () => {
  const tree = renderer
    .create(
        <Router history={browserHistory}>
          <ErrorPage
            errorText={``}
          />
        </Router>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
