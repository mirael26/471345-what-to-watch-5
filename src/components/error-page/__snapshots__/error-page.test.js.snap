// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ErrorPage render correctly 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="main.html"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
  </header>
  <h2
    className="page-title user-page__title"
    style={
      Object {
        "top": "23%",
      }
    }
  >
    
  </h2>
  <div
    className="sign-in user-page__content"
  >
    <div
      className="sign-in__submit"
    >
      <a
        className="sign-in__btn"
        href="/"
        onClick={[Function]}
      >
        Go to the main page
      </a>
    </div>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="main.html"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
